images/scripts/debian.setup

```diff
@@ -116,6 +116,18 @@ if [ "$1" = "debian-testing" ]; then
     rm --verbose -f /etc/apt/sources.list.d/*
     echo 'deb http://deb.debian.org/debian testing main' > /etc/apt/sources.list
     RELEASE=testing
+
+    # HACK: install thin-provisioning-tools from unstable https://bugs.debian.org/1041089
+    echo 'deb http://deb.debian.org/debian unstable main' >> /etc/apt/sources.list
+    cat <<EOF > /etc/apt/preferences.d/unstable
+Package: *
+Pin: release a=unstable
+Pin-Priority: -1
+
+Package: thin-provisioning-tools
+Pin: release a=unstable
+Pin-Priority: 10
+EOF
```

Comment:
Does this mean an `apt upgrade` will also update to unstable? Not that it matters too much as this is hopefully temporary but good to know.